test/crawlers/cheerio_crawler.test.js

```diff
@@ -638,27 +639,22 @@ describe('CheerioCrawler', () => {
             await crawler.run();
             delete process.env[ENV_VARS.PROXY_PASSWORD];
 
-            utilsRequestMock.verify();
-            // expect(proxies).to.have.lengthOf(1);
+            utilsRequestMock.restore();
             expect(proxies[0]).toEqual(proxy);
             expect(proxies[1]).toEqual(proxy);
             expect(proxies[2]).toEqual(proxy);
             expect(proxies[3]).toEqual(proxy);
         });
 
-        test('should handleRequestFunction contains proxy object', async () => {
+        test('should handleRequestFunction contains proxyInfo object', async () => {
```

Comment:
Doesn't make sense as an english sentence. Try: 

`handleRequestFunction should expose the proxyInfo object`

Reply:
Now I'm wondering. We should test that it's in the `handlePageFunction`, not in `handleRequestFunction` which is private.

Reply:
Fixed.